src/kernel/kmain.zig

```diff
@@ -137,6 +141,31 @@ fn initStage2() noreturn {
         else => {},
     }
 
+    const kb = keyboard.getKeyboard(0) orelse unreachable;
+    var shift = false;
+    while (true) {
+        if (kb.readKey()) |key| {
+            if (key.released) {
+                if (key.position == keyboard.KeyPosition.LEFT_SHIFT) {
+                    shift = false;
+                }
+                continue;
+            }
+            var char: ?u8 = switch (key.position) {
+                keyboard.KeyPosition.LEFT_SHIFT, keyboard.KeyPosition.RIGHT_SHIFT => blk: {
+                    shift = true;
+                    break :blk null;
+                },
+                keyboard.KeyPosition.Q => if (shift) @as(u8, 'Q') else @as(u8, 'q'),
+                keyboard.KeyPosition.W => if (shift) @as(u8, 'W') else @as(u8, 'w'),
```

Comment:
I guess later, we can add a keymap file that will perform the translation for use. Plus, aren't chars already u8's?

Reply:
Indeed, this was just for testing. I didn't actually intend to include so I'll remove it.